### ISSUE TITLE
Import Modular SCSS and $themeColor variable

### ISSUE THREAD

**Issue by proov** [nolimits4web/swiper#3327]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 5.2.0
* Platform/Target and Browser Versions: macOS 10.15.1
* Live Link or JSFiddle/Codepen or website with isssue: https://codesandbox.io/embed/great-heisenberg-f7uyy

### What you did
Importing all SCSS `node_modules/swiper/components` components to load only what i want. See `_swiper.scss` : https://codesandbox.io/embed/great-heisenberg-f7uyy

Also, i wrote (& commented) a dirty fix, i would avoid using it :D

### Expected Behavior
Navigation arrows should get the $themeColor color.

### Actual Behavior
Navigation arrows stays black.

Thanks 🙌

**Comment by joebentaylor1995** [comment]
This seems more of a styling issue with your code, rather than a swiper problem?

**Comment by joebentaylor1995** [comment]
Ive fixed your code here @proov 

add this to your `swiper.scss` file

```
// This is the fix
.swiper-button-prev,
.swiper-button-next {
  &:after {
    color: $themeColor;
  }
}
```

**Comment by cfxd** [comment]
@proov I'm with you. I feel like the line in `components/core/core.scss` should be interpolated:

```
--swiper-theme-color: #{$themeColor};
```

If I make that change in my src (or use your "dirty" fix) then it works exactly as expected.

**Comment by proov** [comment]
> @proov I'm with you. I feel like the line in `components/core/core.scss` should be interpolated:
> 
> ```
> --swiper-theme-color: #{$themeColor};
> ```
> 
> If I make that change in my src (or use your "dirty" fix) then it works exactly as expected.

Yeah that's what i think too 😅

**Comment by nolimits4web** [comment]
Fixed by #3334
